src/x86/sse.asm

```diff
@@ -129,6 +129,7 @@ cglobal weighted_sse_4x4, 6, 7, 5, \
 
     pshuflw             m0, m1, q3232
     paddd               m0, m1
+    xor                rax, rax
```

Comment:
movd clears the upper 32 bits of the output register, so I believe this xor shouldn't be necessary. Or perhaps it's to avoid a false dependency on the destination register? if so `xor eax, eax` should be less in code size but still also clear the upper 32 bits anyway, and is apparently the recommended zeroing method (https://stackoverflow.com/questions/33666617/what-is-the-best-way-to-set-a-register-to-zero-in-x86-assembly-xor-mov-or-and/33668295#33668295)